packages/report-server/src/reportBuilder/transform/functions/fetchData/query/buildOrganisationUnitParams.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
```

Comment:
Mostly copied from the previous `packages/report-server/src/reportBuilder/query/buildOrganisationUnitParams.ts`